<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>712</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>980</width>
    <height>712</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>980</width>
    <height>712</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ReceiveBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>收件箱</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Logout">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>登出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SentBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>已发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TrashBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>垃圾箱</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ScriptBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>草稿箱</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WriteEmail">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>写邮件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UnreaedBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>未读邮件</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>981</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>191</width>
      <height>101</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>191</width>
      <height>101</height>
     </size>
    </property>
    <property name="text">
     <string>欢迎您使用FlowerMail！</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SettingB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>主题设置</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>231</y>
      <width>711</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="help_btn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QPushButton" name="safe_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>安全设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeUpdate">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>显示时间</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
